src/inc/cfi.h

```diff
@@ -0,0 +1,31 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
```

Comment:
Stephen Toub changed all the header file license agreements yesterday, so you should update this to match the new standard.